### ISSUE TITLE
How to's or tutorials?

### ISSUE THREAD

**Issue by danesparza** [michaelnoonan/RazorSharpEmail#3]
Got a howto or a tutorial anywhere?  I'd love to see how to use this thing.

**Comment by michaelnoonan** [comment]
Yeah, that's a good point Dan. I've just open sourced the library because
myself and few colleagues have found it handy on production projects. :)
